src/electron/main/main.ts

```diff
@@ -41,6 +40,14 @@ const createWindow = () => {
         mainWindow.show();
         enableDevMode(mainWindow);
     });
+
+    electronAutoUpdateCheck
+        .check()
+        .then(() => {
+            console.log('checked for updates');
```

Comment:
is this `console.log` needed?

Reply:
This wont be logged anywhere on the console, so doesn't hurt and its easier to check while developing! So, yes I am inclined to keep this